### ISSUE TITLE
What if i need to submit my app's source code ?

### ISSUE THREAD

**Issue by flazouh** [wdzeng/firefox-addon#5]
<img width="499" alt="Screenshot 2024-08-12 at 00 29 53" src="https://github.com/user-attachments/assets/116c5fd7-532e-45f5-b195-2568f0bed777">

**Comment by flazouh** [comment]
is this supported by the amo api ?

**Comment by wdzeng** [comment]
> is this supported by the amo api ?

Yes, but my action does not have this feature. I am working on this.

**Comment by wdzeng** [comment]
Hi @flazouh, did you want to try `v1.1.0-alpha.1` that supports upload source code file?

```yaml
steps:
  - uses: wdzeng/firefox-addon@v1.1.0-alpha.1
    with:
      addon-guid: your-addon's-guid
      xpi-path: your-addon.zip
      self-hosted: false
      jwt-issuer: ${{ secrets.FIREFOX_JWT_ISSUER }}
      jwt-secret: ${{ secrets.FIREFOX_JWT_SECRET }}
      source-file-path: your-source-file.zip
```

**Comment by flazouh** [comment]
Hey ! I will test it. Interesting though, that I uploaded my code once and now, each time I upload a new version, it gets accepted nearly instantly and I don't need to upload source code.

**Comment by wdzeng** [comment]
Landed on v1.1.

**Comment by flazouh** [comment]
Hi @wdzeng, I haven't had the time to test since my code hasn't been refused once for now and is accepted after 1minute. But hopefully i can test your work soon !! 🎉
